src/builder-main.c

```diff
@@ -528,6 +528,8 @@ main (int    argc,
   builder_context_set_jobs (build_context, opt_jobs);
   builder_context_set_rebuild_on_sdk_change (build_context, opt_rebuild_on_sdk_change);
   builder_context_set_bundle_sources (build_context, opt_bundle_sources);
+  builder_context_set_opt_export_only(build_context, opt_export_only);
+  builder_context_set_opt_mirror_screenshots_url(build_context, opt_mirror_screenshots_url);
```

Comment:
style nit: these parens on the function calls & the others here should have a space after the function name